docs/docs/text/text.md

```diff
@@ -0,0 +1,38 @@
+---
+id: text
+title: Text
+sidebar_label: Text
+slug: /text/text
+---
+
+The text component can be used to draw a simple text.
+The font family and the font size must be specified.
+The fonts available in the canvas are described in [here](/docs/text/fonts).
+
+Images can be draw by specifying the output rectangle and how the image should fit into that rectangle.
```

Comment:
Could this line be removed? Because the "fit-in-rect" functionality is in the paragraph module (scheduled for later)?